README.md

```diff
@@ -43,6 +47,19 @@ If you are upgrading from a previous version of blender2ogre, and having problem
 * [Animations](http://www.youtube.com/watch?feature=player_embedded&v=5oVM0Lmeb68)
 * [Meshmoon: Video and text instructions how to install and use blender2ogre addon](http://doc.meshmoon.com/index.html?page=from-blender-to-meshmoon-part-1)
 
+## Exporting meshes
+To export a blender model: `File Menu > Export > Ogre3D (.scene & .mesh)`. If the selection is greyed out, the select the object to export from the blender Node tree (Scene collections). 
+
+If you have `OGRETOOLS_XML_CONVERTER` set to a `OgreXMLConverter.exe` path, then the export dialogue will display options relevant for the Ogre (v1) mesh format.
+
+If you have `OGRETOOLS_XML_CONVERTER` set to a `OgreMeshTool.exe` path, then the export dialogue will display options relevant for the OgreNext (v2) mesh format. If you do want to export in the OgreNext (v2.*) format, make sure in the `Export dialogue > General Settings > Mesh Export Version` is set to V2. The following combinations of parameters are recommended by the `OgreMeshTool.exe` help screen:
+* Recommended params for modern DESKTOP (with normal mapping): **OgreMeshTool -e -t -ts 4 -O puqs sourcefile [destfile]**
```

Comment:
OgreMeshTool documentation does not belong here. It will get out of sync pretty fast. Please remove this. 

Actually it is out of sync already, as ogre-next does not support GLES2.

Reply:
My intention behind the recommended settings text was to give less knowledgable people a hint what settings they should use as defaults.  I don't know enough beind the scene to make the claim that these are good settings so I wanted to point to a source.
Would you prefer the text shortened to something like:
**The following combinations of arguments are a recommended start point: 
* modern DESKTOP (with normal mapping): **OgreMeshTool -e -t -ts 4 -O puqs sourcefile [destfile]
* modern DESKTOP (without normal mapping): **OgreMeshTool -e -O puqs sourcefile [destfile]**
Or removed entirely?

Reply:
please remove this entirely.

Reply:
I get your point. See my update for this section.